components/automate-ui/src/app/modules/desktop/check-in-time-series/check-in-time-series.component.scss

```diff
@@ -27,6 +10,47 @@
   }
 }
 
+.heading {
+
+  .heading-title-container {
+    display: flex;
+    align-items: center;
+    margin-bottom: .25em;
+
+    .heading-title {
+      color: var(--chef-primary-dark);
+      margin-top: 0;
+      margin-bottom: 0;
+      font-weight: inherit;
+      font-style: inherit;
+      font-size: 18px;
+    }
+
+    chef-icon {
+      margin-left: 4px;
+      font-size: 24px;
+      cursor: help;
+    }
+
+    chef-tooltip {
+      width: 288px;
```

Comment:
How was 288px chosen?

Reply:
it was the width from invision files, which made the text layout match those from the mock up.